app-admin/conserver/conserver-8.2.7_p20250129.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools pam ssl-cert
+
+COMMIT_HASH="290933b4a7964d56f74d2e3c61f7045c5e0d6bfe"
+DESCRIPTION="Serial Console Manager"
+HOMEPAGE="https://www.conserver.com"
+SRC_URI="https://github.com/bstansell/${PN}/archive/${COMMIT_HASH}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${PN}-${COMMIT_HASH}"
+
+LICENSE="BSD BSD-with-attribution"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="freeipmi ipv6 kerberos pam ssl tcpd"
+
+DEPEND="
+	virtual/libcrypt:=
+	freeipmi? ( sys-libs/freeipmi )
+	kerberos? (
+		net-libs/libgssglue
+		virtual/krb5
+	)
+	pam? ( sys-libs/pam )
+	ssl? ( dev-libs/openssl:0= )
+	tcpd? ( sys-apps/tcp-wrappers )
+"
+RDEPEND="${DEPEND}
+	pam? ( sys-auth/pambase )
```

Comment:
I feel like we don't need this? It's implied that pam will need pam configuration and pam itself depends on it.

Reply:
Sure. I was surprised to see config files in rdep.